src/tasks/DockerPseudoterminal.ts

```diff
@@ -22,10 +22,12 @@ export class DockerPseudoterminal implements Pseudoterminal {
     private readonly writeEmitter: EventEmitter<string> = new EventEmitter<string>();
     private readonly cts: CancellationTokenSource = new CancellationTokenSource();
 
+    /* eslint-disable-next-line no-invalid-this */
     public readonly onDidWrite: Event<string> = this.writeEmitter.event;

```

Comment:
Just for my understanding , why does eslint comlains about this in the member variable?

Reply:
It's the use of `this`; the rule doesn't comprehend well setters for properties/fields of classes.

Reply:
Luckily this construct isn't very common.